cypress/integration/unit_tests/process_joined_data_test.js

```diff
@@ -20,48 +20,67 @@ joinedDataPromise.then = (lambda) => lambda({features: [feature]});
 
 describe('Unit test for processed_joined_data.js', () => {
   it('Processes an above threshold block group', () => {
-    const result = processJoinedData(
-        joinedDataPromise, 100 /* scalingFactor */, 0.3 /* povertyThreshold */,
-        0.5 /* damageThreshold */, 0.5 /* povertyWeight */);
-    expect(result).to.be.an('array');
-    expect(result.length).to.equal(1);
-    const returnedFeature = result[0];
-    expect(returnedFeature).to.have.property('geometry', geometryObject);
-    expect(returnedFeature).to.haveOwnProperty('properties');
-    const resultProperties = returnedFeature.properties;
-    // We modify the properties in place.
-    expect(resultProperties).to.equal(featureProperties);
-    expect(resultProperties).to.have.property('BLOCK GROUP', 'block group');
-    expect(resultProperties)
-        .to.have.property(
-            'SCORE', Math.round(100 * (0.5 * ((10 + 5) / 27) + 0.5 * (2 / 4))));
-    assertColorAndOpacity(resultProperties, 135);
+    cy.wrap(processJoinedData(
```

Comment:
Because you're not doing any Cypress commands after this, you can just chain the .then() off of the processJoinedData. In this test file and update_test.js as well.